addons/mail/static/tests/discuss/call/ptt_ad_banner.test.js

```diff
@@ -19,12 +18,15 @@ test("display banner when ptt extension is not enabled", async () => {
     mockGetMedia();
     const pyEnv = await startServer();
     const channelId = pyEnv["discuss.channel"].create({ name: "General" });
-    mockService("discuss.ptt_extension", () => ({
-        ...pttExtensionHookService.start(getMockEnv(), {}),
-        get isEnabled() {
-            return false;
-        },
-    }));
+    mockService("discuss.ptt_extension", (...args) => {
+        const service = pttExtensionHookService.start(...args);
```

Comment:
It happens frequently that we want to essentially reuse the code of the service but only slightly patch for the test. `mockService` is designed to fully mock, so it's not great to hack and make small patch here.

Probably we could make the effort to turn such services into classes and just `patchWithCleanup` rather than `mockService`. Here this was made to convert tests to HOOT for which I avoided to change business code to reduce amount of conflicts to the minimum.